### ISSUE TITLE
Incorrect change of postfix operator causes code to break

### ISSUE THREAD

**Issue by kimbauters** [nicklockwood/SwiftFormat#415]
I define a transposition operator as a postfix operator as follows:
```
postfix operator ′
public postfix func ′ (value: Matrix) -> Matrix {
    return value.transpose()
}
```
Note that this is not a backtick character, but another visually close unicode character.

The expectation would be, especially since this is in the same file, that this operator symbol is henceforth ignored. However, spaces are added transforming
```
return self′.addRow(row: column)′
```
into
```
return self ′ .addRow(row: column)′
```
which breaks the code.

I don't know quite which rule is causing the change to happen. I've tried to disable a few (`operatorfunc` and `ranges`) but the results remain the same.

**Comment by nicklockwood** [comment]
@kimbauters thanks for reporting, I'll try to get this fixed. The rule you need to disable in the meantime is `spaceAroundOperators`.

**Comment by nicklockwood** [comment]
@kimbauters fixed in 0.40.7

**Comment by kimbauters** [comment]
Brilliant, thanks for sorting this out so quickly! I can indeed confirm that now valid Swift code is generated. There is one oddity though: 
```
public postfix func ′ (value: Matrix) -> Matrix {
    return value.transpose()
}
```
In the definition, a space is introduced between the name of the function `′` and the function arguments. It is syntactically correct and compiles, yet feels somewhat odd.
